File: all/abc190/abc190_a/abc190_a.cs
class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int a = int.Parse(t[0]);
    int b = int.Parse(t[1]);
    int c = int.Parse(t[2]);

    while(true)
    {
      if(c == 0)
      {
        if(a == 0)
        {
          Console.Write("Aoki");
          return;
        }
        a--;
      }
      else
      {
        if(b == 0)
        {
          Console.Write("Takahashi");
          return;
        }
        b--;
      }

      c ^= 1;
    }
  }
}

File: all/abc190/abc190_a/abc190_a.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();
    int c = stdIn.nextInt();

    final int INF = 10009;
    for(int i = c; i < INF; i++) {
      if(i % 2 == 0) {
        if(a == 0) {
          System.out.println("Aoki");
          break;
        }

        a--;
      } else {
        if(b == 0) {
          System.out.println("Takahashi");
          break;
        }

        b--;
      }
    }
  }
}